modules/apps/jaggery/mdm/src/mdm/units/login/public/js/login-box.js

```diff
@@ -1,3 +1,37 @@
+
+function utf8_to_b64( str ) {
+    return window.btoa(unescape(encodeURIComponent( str )));
```

Comment:
There is a discussion goin on regarding this on dev mailing list under the subject "[Dev][Encoding Special Characters] Throws an Character Out Of Range exception" .
An Update will be provided accordingly to the outcome of the discussion.